internal/transformations/normalise_path.go

```diff
@@ -13,10 +14,15 @@ func normalisePath(data string) (string, bool, error) {
 		return data, false, nil
 	}
 	clean := filepath.Clean(data)
```

Comment:
same approach as in https://github.com/corazawaf/coraza/pull/1135#pullrequestreview-2265644142 for a `Clean` function.

Reply:
As said in the other issue, the compiler optimizes the if away. Using build tags and seperate files would make this far more complicated.